terraform/templates/ec2_user_data.sh

```diff
@@ -39,3 +39,22 @@ yum -y install ngrep tcpdump perf gdb nmap-ncat strace htop sysstat tc git
 if [ ! -d /usr/local/etc/FlameGraph ] ; then
     git clone --depth 1 https://github.com/brendangregg/FlameGraph /usr/local/etc/FlameGraph
 fi
+
+mkdir -p /opt/logstash
+
+cat > /opt/logstash/logstash.conf <<"EOF"
+input {
+  file {
+    path => "/opt/libra/data/libra.log"
+  }
+}
+
+output {
+  amazon_es {
+      hosts => ["https://search-validator-logging-tzrcylcyokg64ze7vatqjwmuba.us-west-2.es.amazonaws.com"]
```

Comment:
The endpoint should be set from a variable. The variable should be blank here, because external people can run this TF config.